xptifw/CMakeLists.txt

```diff
@@ -16,6 +16,12 @@ endif()
 
 project (xptifw)
 
+foreach(flag_var
```

Comment:
Since my machine died, I have not been able to reproduce the Windows build failures.  To be a part of LLVM builds, the Debug libraries must be of the format **xxxxd**.dll or they will overwrite the release version and vice-versa and they are not ABI compatible on Windows. The logic that is there in 'xpti' may be needed here to ensure that happens, especially when invoked by LLVM builds as the binaries may be placed in the same directory.

Reply:
This piece of code changes static MSVC CRT to Dynamic. So, if there's a `/MTd` flag in command line options, it'll be replaced with `/MDd`. The `d` postfix is preserved, so, it should be fine for debug and non-debug builds.